testing/font-comic-neue/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=font-comic-neue
+_pkgname=comicneue
+pkgver=20160917
+pkgrel=0
+pkgdesc="Comic Neue is a font that fixes the shortcomings of Comic Sans"
+url="http://comicneue.com/"
+arch="noarch"
+license="OFL-1.1"
+depends="fontconfig"
+makedepends=""
+commit="216e1cca320e7e7886c9523f9afbb7972eb7a9f2"
```

Comment:
Non-standard global variables must be prefixed with underscore. Also common name for this is `_gitrev` and it should be placed under `pkgver` to prevent overlooking it when upgrading abuild.